cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -390,41 +391,6 @@ def resource_estimate(
             return resource_estimates
         return resource_estimates[0]
 
-    def aqt_compile(
-        self,
-        circuits: Union[cirq.Circuit, List[cirq.Circuit]],
-        target: str = "aqt_keysight_qpu",
-        atol: Optional[float] = None,
-        gate_defs: Optional[
-            Mapping[str, Union[npt.NDArray[np.complex_], cirq.Gate, cirq.Operation, None]]
-        ] = None,
-        **kwargs: Any,
-    ) -> css.compiler_output.CompilerOutput:
-        """Compiles and optimizes the given circuit(s) for the Advanced Quantum Testbed (AQT) at
-        Lawrence Berkeley National Laboratory.
-
-        Args:
-            circuits: The circuit(s) to compile.
-            target: String of target AQT device.
-            atol: An optional tolerance to use for approximate gate synthesis.
-            gate_defs: An optional dictionary mapping names in qtrl configs to operations, where
-                each operation can be a unitary matrix, `cirq.Gate`, `cirq.Operation`, or None. More
-                specific associations take precedence, for example `{"SWAP": cirq.SQRT_ISWAP,
-                "SWAP/C5C4": cirq.SQRT_ISWAP_INV}` implies `SQRT_ISWAP` for all "SWAP" calibrations
-                except "SWAP/C5C4" (which will instead be mapped to a `SQRT_ISWAP_INV` gate on
-                qubits 4 and 5). Setting any calibration to None will disable that calibration.
-            kwargs: Other desired aqt_compile options.
-
-        Returns:
-            Object whose .circuit(s) attribute contains the optimized circuits(s). If qtrl is
-            installed, the object's .seq attribute is a qtrl Sequence object containing pulse
-            sequences for each compiled circuit, and its .pulse_list(s) attribute contains the
-            corresponding list(s) of cycles.
-        """
-
-        _validate_cirq_circuits(circuits)
-        return self._aqt_compile(circuits, target=target, atol=atol, gate_defs=gate_defs, **kwargs)
-
     def aqt_compile_eca(
         self,
         circuits: Union[cirq.Circuit, Sequence[cirq.Circuit]],
```

Comment:
I think the type for `circuits` here should also be updated to: 
```suggestion
        circuits: Union[cirq.Circuit, List[cirq.Circuit]],
```
following line 456

Reply:
good catch, updated for consistency (it looks like we use `List[Circuit]` in input annotations everywhere in qss/css)

however - i think really we should be using `Sequence` for all of these input annotations. in general input annotations should be as general as possible and output annotations should be as specific as possible, and these functions should work with any `Sequence` type (e.g. tuples) instead of just lists. will open a follow up pr to change this